phpBB/styles/prosilver/template/navbar_footer.html

```diff
@@ -16,16 +16,23 @@
 			{% apply spaceless %}
 			<span class="crumb">
 				<a href="{U_INDEX}" data-navbar-reference="index">
-					<!-- IF not U_SITE_HOME --><i class="icon fa-home fa-fw" aria-hidden="true"></i><!-- ENDIF --><span>{L_INDEX}</span>
+					<!-- IF not U_SITE_HOME -->{{ Icon('iconify', 'fa:home', lang(L_INDEX), false) }}<!-- ENDIF -->
```

Comment:
Now you remove the text aswell when there is a 'Main website'. The check is only if an icon should be displayed.
```suggestion
					{% if U_SITE_HOME %}<span>{{ lang('INDEX') }}</span>{% else %}{{ Icon('iconify', 'fa:home', lang('INDEX'), false) }}{% endif %}
```